public/scripts/ssooverride.js

```diff
@@ -83,34 +92,62 @@
         }, false);
       });
 
-      var thimbleAuth = new WebmakerLogin({
-        csrfToken: csrf,
-        showCTA: false
-      });
+      var basicQuery = "";
+
+      basicQuery += "?client_id=" + oauthClientId;
+      basicQuery += "&response_type=code";
+      basicQuery += "&scopes=user email";
 
+      // Signup login flow
       joinEl.on('click', function() {
-        thimbleAuth.create();
+        var state = uuid.v4();
+        var oauthRoute = "/login/oauth/authorize";
+
+        var query = hostname + oauthRoute + basicQuery;
+        query += "&state=" + state;
+        query += "&action=signup";
+
+        setStateCookie(state);
+
+        window.location = query;
       });
+
+      // Login flow
       loginEl.on('click', function() {
-        thimbleAuth.login();
+        var state = uuid.v4();
+        var oauthRoute = "/login/oauth/authorize";
+
+        var query = hostname + oauthRoute + basicQuery;
+        query += "&state=" + state;
+        query += "&action=signin";
+
+        setStateCookie(state);
+
+        window.location = query;
       });
       logoutEl.on('click', function() {
-        thimbleAuth.logout();
-      });
+        // Logout flow
+        var oauthRoute = "/logout";
 
-      // Attach event listeners!
-      thimbleAuth.on('login', enable);
-      thimbleAuth.on('logout', disable);
-      thimbleAuth.on('verified', function(user) {
-        if (user) {
-          enable(user);
-        } else {
-          disable();
-        }
+        var query = hostname + oauthRoute;
+        query += "?client_id=" + oauthClientId;
+
+        window.location = query;
       });
 
-      // Default state is signed-out
-      disable();
+      var thimbleCookie = cookies.get("mozillaThimble");
+
+      if (!thimbleCookie) {
```

Comment:
The `cookie-session` docs say that it creates a session per visitor, but during my testing this cookie hadn't been created when this first ran. I don't get why...